shadows/framework/src/main/java/org/robolectric/shadows/ShadowBiometricManager.java

```diff
@@ -1,7 +1,9 @@
 package org.robolectric.shadows;
 
 import static android.Manifest.permission.USE_BIOMETRIC;
+import static android.hardware.biometrics.BiometricManager.BIOMETRIC_ERROR_NONE_ENROLLED;
 import static android.hardware.biometrics.BiometricManager.BIOMETRIC_ERROR_NO_HARDWARE;
+import static android.hardware.biometrics.BiometricManager.BIOMETRIC_ERROR_SECURITY_UPDATE_REQUIRED;
```

Comment:
This field is added from R, but this shadow's `minSdk` is Q, maybe it will cause building error? Let's run tests with our CI's Q-compatibility later.

Reply:
@charlesng Could you remove this importing and use full qualify in related test method?

Reply:
@utzcoz Update, not sure if this change matches your expectation. Btw, just curious, why we only need full qualify import statement in test method? 🤔

Reply:
In many time, we can import static fields. In this occasion, I recommend to move BIOMETRIC_ERROR_SECURITY_UPDATE_REQUIRED to full qualify import to avoid compatibility problem because it is added from R, but this shadow's default minSdk is Q.